stories/Molecules/DownloadRow/DownloadRow.jsx

```diff
@@ -6,15 +6,15 @@ import { Heading } from '../../Atom/Typography/Heading/Heading';
 export const DownloadRow = ({
   title, subtitle, value, ...args
 }) => (
-  <div className="checkbox-container">
-    <div className="checkbox-container__main">
-      <div className="checkbox-container__data">
+  <div className="download-row">
+    <div className="download-row__main">
+      <div className="download-row__data">
         <Heading type="6" label={title} />
         <span>{subtitle}</span>
       </div>
       {args.type === 'Checkbox' && <Checkbox value={value} /> }
-      {args.type === 'Download' && <span className="checkbox-container__chevron download-icon" /> }
-      {args.type === 'External_link' && <span className="checkbox-container__chevron external-link" /> }
+      {args.type === 'Download' && <a href="#"><span className="download-row__chevron download-icon" /> </a>}
+      {args.type === 'External_link' && <a href="#"><span className="download-row__chevron external-link" /></a>}
```

Comment:
@pallavikadam-srijan Kindly remove the `download-row__chevron` class and use parent selector to style the icons here. 
Example:
```
.download {
  &-row & {
    &-icon { ... }
  }
}
```
The above will compile to:
```
.download-row .download-icon { ... }
```